sys/Makefile.include

```diff
@@ -102,4 +102,7 @@ endif
 
 ifneq (,$(filter newlib,$(USEMODULE)))
     include $(RIOTBASE)/sys/newlib/Makefile.include
-endif
\ No newline at end of file
+endif
+
+# add libc header overrides to head of include paths
+INCLUDES := -I$(RIOTBASE)/sys/include/libc $(INCLUDES)
```

Comment:
> Why is it prepended and if this is required: how do you make sure that nothing else changes this again?

Hm, it's a standard header, but actually it shouldn't matter where in
the path the directory ends up.